lib/XPathAbstract.php

```diff
@@ -532,7 +532,7 @@ protected function formatItemCategories($value) {
 	 */
 	protected function cleanImageUrl($imageUrl)
 	{
-		$result = preg_match('~(?:http(?:s)?:)?[\/a-zA-Z0-9\-_\.]+\.(?:jpg|gif|png|jpeg|ico){1}~', $imageUrl, $matches);
+		$result = preg_match('~(?:http(?:s)?:)?[\/a-zA-Z0-9\-_\.]+\.(?:jpg|gif|png|jpeg|ico|mp3){1}~', $imageUrl, $matches);
```

Comment:
looks like cleanImageUrl should be renamed to cleanMediaUrl. Same about parameter.